### DISCUSSION TITLE
Marzban on blocked server

### DISCUSSION THREAD

**Discussion by guvanch7** [Gozargah/Marzban#675] [Q&A]
Hello. Sorry for such a stupid question, the thing is that I am new to this field. 

The question is this. Is it possible to install Marzban on a server blocked by ISP and configure it to work?

Why I'm asking, I recently saw a guy somehow managed to get around the blockage.

Please advise me on the options.

**Comment by danilka38370** [comment]
это можно сделать если ты привяжешь ip к домену на cloudflare и включишь режим прокси но правда надо будет выпустить сертификат для домена, но в принципе это можно сделать и пользоваться заблокированными адресами и даже ноды могут быть заблокированны но ими можно пользоваться

**Reply by guvanch7** [reply]
Да, уже сделал) Спасибо
